app/services/course/assessment/submission/worksheet_service.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+class Course::Assessment::Submission::WorksheetService < Course::Assessment::SubmissionService
```

Comment:
Line is too long. [94/80]<br>Use nested module/class definitions instead of compact style.